subjects/mobile-dev/bloc-counter/audit/README.md

```diff
@@ -1,13 +1,17 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
 ###### Does the app run without any crashes?
 
-###### Can you increment counter?
+###### Can you increment the counter by tapping the "+" button?
 
-###### Can you decrement counter?
+###### Can you decrement the counter by tapping the "-" button?
 
-###### Is the BLoC pattern implemented properly? Do you see blockobserver, blockprovider, blockbuilder explicitly?
+###### Is the BLoC pattern implemented properly? Can you see the BlocObserver, BlocProvider, and BlocBuilder being used explicitly in the code?
 
-###### Are incrementing and decrememting implemented using BLoC pattern?
+###### Are the increment and decrement operations implemented using the BLoC pattern, with the CounterBloc class managing the state of the counter?
```

Comment:
In the subject, decrementing is described as a bonus

Reply:
done